packages/client/src/runtime/RequestHandler.ts

```diff
@@ -263,6 +277,26 @@ export class RequestHandler {
     return unpack({ document, data, path: getPath })
   }
 
+  applyResultExtensions({ result, modelName, args, extensions }: ApplyExtensionsParams) {
```

Comment:
Do you think it makes sense to keep these together in `core/extensions`? I'm planing to park `applyQueryExtensions` there